src/vendor/wordpress/styles/_core-blocks.scss

```diff
@@ -0,0 +1,233 @@
+   * generate two different elements, a file-link and a file-button and we
+   * want to style both of them with the same visual appearance and
+   * characteristics. We could easily have chosen `:not(.wp-block-file__button)`
+   * but opted for less characters.
+   *
+   * 1. Prevents default WP styling from modifying buttons opacity on hover.
+   */
+  .wp-block-file__button,
+  * {
+    @include button.default;
+    opacity: inherit !important; /* 1 */
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying button text color on hover
+   */
+  &.c-button--secondary {
+    .wp-block-file__button,
+    * {
+      @include button.secondary;
+      color: inherit !important; /* 1 */
+    }
+  }
+
+  /**
+   * 1. Prevents default WP styling from modifying button text color on hover
+   */
+  &.c-button--tertiary {
+    .wp-block-file__button,
+    * {
+      @include button.tertiary;
+      color: inherit !important; /* 1 */
+    }
+  }
+}
+
+/**
+ * Gutenberg block: Image
+ * Styles for images inserted via Gutenberg blocks.
+ *
+ * 1. Since we are applying `margin-top` to CMS content via our o-rhythm
+ *    pattern, the default `margin-bottom` value on Gutenberg image blocks is
+ *    unnecessary, so we remove it here.
+ */
+.wp-block-image {
+  margin-bottom: 0; /* 1 */
+  position: relative;
+
+  /**
+   * Setting a width of `100%` prevents resized and centered images from
+   * breaking layout when placed after a right-aligned image in a post.
+   */
+  &.aligncenter {
+    width: 100%;
+  }
+
+  /* Since we are applying `margin-top` to CMS content via our o-rhythm
+   * pattern, the default `margin-bottom` value on Gutenberg figcaption
+   * elements is unnecessary, so we remove it here.
+   */
+  figcaption {
+    margin-bottom: 0;
+  }
+}
+
+/**
+ * Gutenberg block: Table
+ * Applies our pattern library styles to tables generated via Gutenberg blocks.
+ */
+
+/**
+ * Default table
+ */
+.wp-block-table table td,
+.wp-block-table table th {
+  @include table.t-container;
+}
+
+.wp-block-table table th {
+  @include table.t-heading;
+}
+
+.wp-block-table table thead {
+  @include table.t-head;
+}
+
+.wp-block-table table tfoot {
+  @include table.t-foot;
+}
+
+.wp-block-table table caption {
+  @include table.t-caption;
+}
+
+/**
+ * Striped table
+ */
+.wp-block-table.is-style-stripes {
+  border-bottom: 0;
+
+  & tbody tr:nth-of-type(2n - 1) {
+    @include table.t-striped;
+  }
+}
+
+/**
+ * Numeric table
+ */
+.wp-block-table.c-table--numeric table {
+  @include table.t-numeric;
+
+  & th {
+    text-align: right;
+  }
+}
+
+/**
+ * Quote and Pull-quote
+ * Styles for quotes and pull-quotes
+ *
+ * 1. Makes the line-height more consistent with existing quote styles.
+ * 2. The quote itself defaults to being italic, this straightens out the quote
+ * 3. The pull-quote isn't italic at all, so for consistency, the cite is
+ * italicized.
+ * 4. The quote is a little squished on small screens so the max width
+ * is increased.
```

Comment:
I might be missing it but I don't explicitly see what this comment refers to (e.g. I don't see `/* 4 */`). Did I miss it? 🤔

Reply:
I'm not sure. This code was moved in this PR but not created for it.

@AriannaChau Do you recall?

Reply:
`
.wp-block-pullquote.is-style-solid-color blockquote { 
@media (min-width: breakpoint.$xs) {    max-width: 80%;  }  
@media (min-width: breakpoint.$m) {    max-width: 60%;  }
}
`
I'm pretty sure it's this block of code but I forgot to add the `/* 4 */` 😅

Reply:
@gerardo-rodriguez I've reformatted this section so that comments precede their associated chunks instead of being spread out from one block comment amongst multiple selectors.